core/src/main/java/org/apache/calcite/sql/type/OperandTypes.java

```diff
@@ -369,6 +369,49 @@ private boolean hasFractionalPart(BigDecimal bd) {
         }
       };
 
+  /**
+   * Operand type-checking strategy type must be a numeric non-NULL
+   * literal in the range 0 - 1.
```

Comment:
inclusive?

Reply:
Yes. thats why i used the name "unit interval".